parser/src/main/java/com/plhal/ares/parser/ParserRepositoryImpl.java

```diff
@@ -52,23 +56,26 @@ public Firma najdiFirmu(String ico) {
 
         } catch (ParserConfigurationException e) {
 
-            e.printStackTrace();
+            log.error("***** ParserConfigurationException occured: ", e);
             return null;
         } catch (SAXException e) {
 
-            e.printStackTrace();
+            log.error("***** SAXException occured: ", e);
             return null;
         } catch (IOException e) {
 
-            e.printStackTrace();
+            log.error("***** IOException occured: ", e);
```

Comment:
tady to stejne, ale protoze jsou vsechny catch bloky stejne, tak bych je rad videl v jednom. Muzete prosim upravit? Vyvojar by se mel drzet pravidla DRY.